modules/databend/src/main/java/org/testcontainers/databend/DatabendContainer.java

```diff
@@ -0,0 +1,124 @@
+package org.testcontainers.databend;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Testcontainers implementation for databend.
+ * <p>
+ * Supported image: {@code datafuselabs/databend}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>Database: 8000</li>
+ * </ul>
+ */
+public class DatabendContainer extends JdbcDatabaseContainer<DatabendContainer> {
+
+    private static final String NAME = "databend";
```

Comment:
NAME is not accessible

```suggestion
    static final String NAME = "databend";
```